src/components/ContactsList/ContactRow.jsx

```diff
@@ -142,12 +68,16 @@ const ContactRow = props => {
       familyName: ""
     }
   } = props.contact;
+  const stopPropagationAndSelect = e => {
+    e.stopPropagation();
```

Comment:
On ne sait pas pourquoi on a besoin de `stopPropagation` ici. C'est plutôt la responsabilité du composant qui crée l'`<input />` ou le `<form />`.

Reply:
Je suis pas trop d'accord. On stoppe la propagation parce qu'il y a un `onClick` sur l'évènement parent et que les 2 entrent en conflit. La propagation doit être arrêtée au moment ou elle commence à poser problème, pas de façon pré-emptive sur un element enfant.

Reply:
I add rendering tree here, for more conveniance:

```jsx
    <div className="contact" onClick={props.onClick}>
       {props.selection && (
         <ContactSelection
           selected={props.selection.selected}
           selected={props.selection.selected}
           onSelect={stopPropagationAndSelect}
         />
       )}
    </div>
```

Oh... I see. We `stopPropagation()` here because we created a tree with mutltiple click listeners. So you might be right. But `<ContactSelection />` may not bubble its event neither, maybe we shall add the `stopPropagation()` on both places.

Reply:
D'instinct, j'aurais tendance à dire la même chose qu'@enguerran : que le `stopPropagation` devrait être géré par `ContactSelection`, mais j'avoue ne pas avoir de faits ou références précises pour étayer mes propos, c'est juste du pif

Reply:
Partons la dessus alors — disons que `onSelect` est appelé pour la sélection, pas au clic.
Sur le principe, `ContactSelection` pourrait être utilisé dans un autre contexte ou l'évènement original a une importance, mais en pratique je ne pense pas que ca arrivera ici.